api/_mixins/HTMLHyperlinkElementUtils__HTMLAreaElement.json

```
@@ -132,7 +132,7 @@
               "version_added": "14"
             },
             "safari": {
-              "version_added": "1"
+              "version_added": "3"
```

```suggestion
              "version_added": "1"
```

Source: https://trac.webkit.org/browser/webkit/trunk/WebCore/khtml/ecma/kjs_html.cpp?rev=4#L729